README.md

```diff
@@ -32,39 +32,30 @@ This version points to the HEAD of the `main` branch and will always fetch the l
 
  ## Architecture
 
-The SDK needs to be initialized with an environment. This allows for different verification rules per environment or other environment specific settings.
+The SDK needs to be initialized with an environment. This allows for different verification rules per environment or other environment-specific settings.
 
 After initialization the following pipeline should be used:
 
 1) Decode the base45 and prefixed string to retrieve a Digital Covid Certificate
 
-2) Verify the signature of the Certificate
-
-3) Check the revocation list. Currently always returns a valid `ValidationResult`
-
-4) Check for rules specific to countries such as validity of vaccines or tests
+2) Verify the Certificate by calling the `.check` method. Internally this verifies the signature, revocation status and national rules
 
 All these checks check against verification properties that are loaded from a server. These returned properties use a property to specify how long they are valid (like `max-age` in general networking). With the parameter `forceUpdate`, these properties can be forced to update.
 
-### Decoding
-```swift
-public func decode(encodedData: String) -> Result<DGCHolder, CovidCertError>
-```
+CovidCertificateSDK offers a `Verifier` and `Wallet` namespace these have to be used according to the purpose of the app. 
```

Comment:
Methods in the Wallet namespace must only be used by the official COVID Certificate App.